src/ResourceManager/Insights/Commands.Insights/ActivityLogs/GetAzureRmLogCommand.cs

```diff
@@ -104,7 +104,8 @@ public class GetAzureRmLogCommand : LogsCmdletBase
         /// <returns>The query filter with the conditions for particular parameters added</returns>
         protected override string ProcessParticularParameters(string currentQueryFilter)
         {
-            this.SetMaxEventsIfPresent(currentQueryFilter, this.MaxEvents);
+            WriteWarning("Parameter name change: The parameter plural names for the parameters will be deprecated in May 2018 in favor of the singular versions of the same names.");
```

Comment:
@gucalder please update the warning message to include the name of the cmdlet - this will help users running this cmdlet in a script easily find which cmdlet the warning message is coming from. Please do this for all warnings in cmdlets.

Reply:
Done. I am including the cmdlet name in all warning messages.